src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -64,6 +65,7 @@ public void Add(Cookie cookie)
             {
                 throw new ArgumentNullException(nameof(cookie));
             }
+            ++m_version;
```

Comment:
> Isn't it only used to determine whether an item has been added during enumeration of the collection?

Correct but we need the field for correct Desktop <--> Core serialization in FormatterAssemblyMode.Full. I don't really care if we remove it or not, I just don't think that removing the increment operation gives us noteworthy performance benefits. I would just stick with the netfx behavior...